### ISSUE TITLE
Very large syntax-highlighted files are slow to work with

### ISSUE THREAD

**Issue by waddlesplash** [KapiX/Koder#47]
e.g. generate a 3000+-line diff by doing `git diff HEAD~20 >out.diff` in the Haiku repo, and try to use the resulting diff in Koder. It will spike to 100% CPU quite often as you page up/down/etc. through the file. Notepad++ handles such files perfectly fine, so the issue it seems is not with Scintilla.

**Comment by KapiX** [comment]
Yup, it's trailing space highlighting. Indicators are pretty slow, I should probably check whole file and then diff only the visible portion when editing. Right now it changes every time screen updates (which is pretty much all the time). Try disabling it and see if it's better.
